lib/prometheus/business_collector.rb

```diff
@@ -88,9 +88,7 @@ def type
 
   def collect
     @total_accounts.observe Account.count
-    client_accounts = Account.distinct.where(
-      id: User.select(:account_id).where.not('email LIKE ?', '%redhat%')
-    )
+    client_accounts = Account.distinct.where.not(is_internal: true)
```

Comment:
Accounts with NULL/nil are not returned. NULL is default and set on existing accounts.

```
> Account.distinct.where.not(is_internal: true)
  Account Load (1.6ms)  SELECT DISTINCT "accounts".* FROM "accounts" WHERE "accounts"."is_internal" != $1  [["is_internal", true]]
=> []

> Account.where(is_internal: nil).count
   (1.5ms)  SELECT COUNT(*) FROM "accounts" WHERE "accounts"."is_internal" IS NULL
=> 4
```

Distinct might be redundant since it is now unique by account_number and it does not matter when used in the subqueries.